tests/data_types.py

```diff
@@ -49,6 +49,13 @@
     }
 }
 
+BLOB_ID = {
+    '_id': {
+        'type': 'string',
+        'pattern': HASH_PATTERN
```

Comment:
I think this will change once multihash lands but we can update it then I suppose.